### ISSUE TITLE
Versioning of plugins vs CLI

### ISSUE THREAD

**Issue by Daniel15** [babel/babili#139]
Is the CLI versioned separately to the plugins? As an example, `babel-plugin-minify-dead-code-elimination` was changed more recently than `babili`, yet `babel-plugin-minify-dead-code-elimination` in the repo has a version number of 0.0.1 while `babili` has 0.0.4.

**Comment by hzoo** [comment]
Right now we are using independent mode from lerna so yes

**Comment by Daniel15** [comment]
So are there two separate version numbers (one for plugins and one for the CLI), or does every plugin have its own separate version number?

For babel-standalone, how should I version the Babili releases? Right now for Babel release I'm getting the maximum version number of all the `babel-*` packages, but maybe that won't work for Babili if everything is versioned independently.

**Comment by hzoo** [comment]
https://github.com/lerna/lerna#how-it-works Right now each package in `packages/` has it's own number

**Comment by Daniel15** [comment]
Yeah I think it's advantageous to have one version number for Babili. Every plugin having its own version number is a bit confusing :smile:

**Comment by vikramaroskar** [comment]
Not much expertise on babel* yet. but in general having separate versioning does have its own advantages, and I prefer this in general. My 2 cents. There is a case for updating versions synchronously. 
A small caveat: A similar thing happening with Angular (they're dropping v3 and jumping to 4)
